ui/src/alerts/components/AlertsTableRow.tsx

```diff
@@ -0,0 +1,124 @@
+import React, {PureComponent} from 'react'
+import {Link} from 'react-router'
+
+import classnames from 'classnames'
+
+import {ALERTS_TABLE} from 'src/alerts/constants/tableSizing'
+
+interface Props {
+  sourceID: string
+  name: string | null
+  level: string | null
+  time: string | null
+  host: string | null
+  value: string | null
+}
+
+const {colName, colLevel, colTime, colHost, colValue} = ALERTS_TABLE
+
+class AlertsTableRow extends PureComponent<Props> {
+  public render() {
+    return (
+      <>
+        {this.nameCell}
+        {this.levelCell}
+        {this.timeCell}
+        {this.hostCell}
+        {this.valueCell}
+      </>
+    )
+  }
+  private get nameCell(): JSX.Element {
+    const {name} = this.props
+
+    return (
+      <div
+        className="alert-history-table--td"
+        style={{width: colName}}
```

Comment:
should this be a part of a class instead of an inline style?

Reply:
not sure -- that's how it was before, just moved it over. good question tho!